pom.xml

```diff
@@ -171,6 +169,22 @@
                 <scala.version>${scala_2.12.version}</scala.version>
             </properties>
         </profile>
+        <profile>
+            <id>spark-2</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
I think that if you turn on profile `deploy` without the `spark-2` or `spark-3`, `spark-2` will be turned off. right? Or do I misunderstand?

Reply:
Yes that's correct. My main motivation for the `activeByDefault` flag was to ensure `mvn clean package` works without any profiles explicitly set